api/entrypoint.sh

```diff
@@ -8,6 +8,8 @@ python manage.py collectstatic --clear --noinput || exit 1
 
 # migrate database
 python manage.py migrate || exit 1
+python manage.py makemigrations eid_server || exit 1
```

Comment:
`makemigrations` should only be called during development time. That is, please call makemigrations on your dev machine, have the migrations generated and add them to this repository.